dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.mpGraphQL1.0-metrics2.3.feature

```diff
@@ -2,7 +2,7 @@
 symbolicName=com.ibm.websphere.appserver.mpGraphQL1.0-metrics2.3
 visibility=private
 IBM-Provision-Capability: \
-  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.mpGraphQL-1.0))", \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.mpGraphQL-1.0)(osgi.identity=io.openliberty.mpGraphQL-2.0)))", \
   osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.mpMetrics-2.3))"
```

Comment:
what about mpMetrics-3.0 and mpMetrics-4.0?

Reply:
added